projects/canopy/src/lib/card/card.component.scss

```diff
@@ -4,6 +4,118 @@
   display: block;
   @include lg-card;
 
+  &.lg-card--promotion {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+
+    .lg-card-hero-img + .lg-card-content {
+      padding: var(--space-lg);
+      padding-top: var(--space-md);
+    }
+  }
+
+  &.lg-card-orientation {
```

Comment:
Done on the moving part. :)
Not sure on the BEM rules on this one, I think we can leave it as is as it is similar to 'lg-padding-left' or 'lg-padding-bottom' and not sure if we should use '-' or '--'. I would leave it as is but can change it if you think it adheres more to the BEM principles using '--' instead. :)

Reply:
Thanks Alex, having thought about it, I think the BEM convention with `--` is a good idea because currently the orientation class is only applied to the existing card component, and it also has `lg-card` as a prefix. So in that sense it is a modifier class for cards, whereas `lg-padding` is utility class for any block element. Perhaps if in future if we made it more generic, then it could be called simply `lg-orientation`.

Reply:
Yea, makes sense. Done and changed `.lg-card-orientation` to `.lg-card--orientation` :)